spec/requests/api/rest/admin/codec_group_codecs_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+RSpec.describe Api::Rest::Admin::CodecGroupsController, type: :request do
+  include_context :json_api_admin_helpers, type: :'codec-groups'
+
+  describe 'GET /api/rest/admin/codec-groups/{codec_group_id}/codecs' do
```

Comment:
Api::Rest::Admin::CodecGroupCodecResource does not have it's collection endpoint. it only used as include for Api::Rest::Admin::CodecGroup

Reply:
pagination not neede